dist/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS Capstone</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+<script defer src="main.js"></script></head>
+<body>
+    <header>
+        <a href="#">
+            <span class="material-icons">color_lens</span>
+            <p>Art central</p>
+        </a>
+        <a href="#">Pieces <p id="counter">(#)</p> </a>
+        <a href="#">About</a>
+    </header>
+    <section>
+        <ul id="works"></ul>
+    </section>
+    <footer>Created by Alex Gamino under MIT licence</footer>
+    <!-- <div id="overlay"></div>
+    <div id="modal">
+        <img src="#" alt="artwork">
+        <button id="close">&times;</button>
+        <h3>Artwork</h3>
+        <div id="info">
+            <p>by Artist</p>
+            <p>medium</p>
+        </div>
+        <div id="comments">
+            <h4>Comments (num)</h4>
+            <ul>
+                <li>
+                    <p>Comment</p>
+                </li>
+                <li>
+                    <p>Comment</p>
+                </li>
+                <li>
+                    <p>Comment</p>
+                </li>
+            </ul>
+        </div>
+        <div id="addComment">
+            <h4>Add a comment</h4>
+            <form action="post">
+                <input type="text" id="user" name="user" placeholder="Your name">
+                <input type="text" id="comment" name="comment" placeholder="Your insights">
+                <button type="submit" id="addButton">comment</button>
+            </form>
+        </div>
+    </div>
+    </div> -->
```

Comment:
- [ ] You should keep your code clean. Please remove commented-out code blocks. You can use git history to see removed code blocks.

Reply:
I add this comment only to this file to not pollute the review section with multiple similar comments. You have commented-out code in multiple files, please be sure you removed all of them.